src/main/java/shop/service/impl/UserServiceImpl.java

```diff
@@ -75,6 +77,14 @@ public void deleteUser(User user) {
         return userDAO.findUser(email);
     }
 
+    public static Optional<User> getLoggedUser() {
```

Comment:
that's not a good approach.

Reply:
Let me know if u have any trouble to make it, I'll help u. 
If you want to save info about logged user we need to make it in some other way